work_with_shared_resources/using-shared-resource-csi-driver.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="shared-resource-csi-driver"]
+= Shared Resource CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: shared-resource-csi-driver
+
+toc::[]
+
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource Container Storage Interface (CSI) Driver in {builds-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {ocp-product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: 
```

Comment:
The CSI driver can be used with BuildConfigs, Shipwright Builds, Tekton Pipelines, and any other Kubernetes resource that supports CSI volumes. 

@adambkaplan could you confirm?

Reply:
Yes, that is why we chose to use CSI as the foundation for this feature. Any Kubernetes object that uses the CSI Volume Type can use the Shared Resource CSI driver. See upstream [doc](https://kubernetes.io/docs/concepts/storage/volumes/#csi) for more info.

Reply:
Tekton/OpenShift Pipelines doesn't expose the Kubernetes `Volume` API directly - it has a similar concept called "Workspaces" that does expose the underlying `csi` volume type. See [Tekton workspaces doc](https://tekton.dev/docs/pipelines/workspaces/#csit-builds)